### PR TITLE
Fix create proposals

### PR DESCRIPTION
**Pull request by cedricmessiant**
Fixes DEVAS-1339

### CODE REVIEW

File: assembl/tests/fixtures/vote_session.py [hunk 1]
```diff
@@ -29,13 +29,24 @@ def vote_session(request, test_session, discussion, timeline_vote_session,
...
+    identifier = 'voteSession{}'.format(vote_session.id)
+    short_title = u'Phase {}'.format(identifier)
+    root_thematic = Thematic(
+        discussion_id=discussion.id,
+        title=LangString.create(short_title, "en"),
+        identifier=identifier,
+        hidden=True)
+    discussion.root_idea.children.append(root_thematic)
+    test_session.flush()
+
```
Comment (vincentfretin) [inline assembl/tests/fixtures/vote_session.py]: Why don't you use the create_root_thematic function here?

File: assembl/graphql/vote_session.py [hunk 2]
```diff
@@ -775,7 +781,8 @@ def mutate(root, args, context, info):
             identifier = 'voteSession{}'.format(vote_session_id)
             root_thematic = get_root_thematic_for_phase(discussion, identifier)
             if root_thematic is None:
-                root_thematic = create_root_thematic(discussion, identifier)
+                raise Exception(
+                    "There is no root thematic for this vote session.")
```
Comment (vincentfretin) [inline assembl/graphql/vote_session.py]: You will have an issue on prod if a vote session was already created and no proposals created yet. You should maybe create an alembic step to create the root thematic for existing vote session?

File: assembl/tests/fixtures/vote_session.py (hunk 1 above)
Comment (cedricmessiant) [reply assembl/tests/fixtures/vote_session.py]: there was a circular import issue

File: assembl/graphql/vote_session.py (hunk 2 above)
Comment (cedricmessiant) [reply assembl/graphql/vote_session.py]: No vote session has been created in prod yet. I will check with Fred if this is an issue

File: assembl/tests/fixtures/vote_session.py (hunk 1 above)
Comment (vincentfretin) [reply assembl/tests/fixtures/vote_session.py]: Even if you put the import here locally?

Comment (cedricmessiant) [reply assembl/tests/fixtures/vote_session.py]: it's good, thanks

**Pull request merged**